docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Getting started
+order: 0
+slug: getting-started
+lang: en
+layout: tutorial
+permalink: /future/getting-started
+---
+
+# Getting started <span class="label-beta">BETA</span>
+
+<div class="section-content">
+This guide will walk you through introducing you to the [next-generation platform](/bolt-js/future/getting-started#next-gen). It will also cover how to get started with your next-generation platform experience with Bolt for Javascript by setting up the Slack CLI and also installing the required dependencies.
+</div>
+
+---
+### What is the next-generation platform? {#next-gen}
```

Comment:
@ashleymcm What do you think about emphasizing that we're in beta at the moment? I know we have the beta label over the whole section, but it might be a good idea to have a sentence with some caveat language about Open Beta and how the product will change with feedback from testers. 

Looking at this from api.slack.com/future for reference. We can also include the feedback link to direct bolt users to the feedback form. 

<img width="1038" alt="Screen Shot 2022-09-14 at 9 58 04 AM" src="https://user-images.githubusercontent.com/55667998/190216601-ad45f7c8-5ed0-46bc-86f7-e18b2da08cab.png">

Reply:
While we're at it... thoughts on adding extra redundancy with a specific page for Feedback that directs to the survey?

Reply:
sounds good! I can add that in right now :D